### ISSUE TITLE
some problems.

### ISSUE THREAD

**Issue by dreamercv** [opencv/openvino_training_extensions#126]
### problems
the losses  were:
* loss:     45.429007072448734
l want to konw ,why the losses is so big.

**Comment by AlexanderDokuchaev** [comment]
@Daniil-Osokin

**Comment by Daniil-Osokin** [comment]
I would say that the losses are quite small, for our model they about twice bigger. Also not sure, if `pycocotools` works properly with 25 keypoint model. To compute the [metric](http://cocodataset.org/#keypoints-eval) they use some hardcoded constants, which control possible variation in coordinates for each type of keypoint, so computed accuracies may be wrong. There can be numerous reasons of these results, you can start from comparison of obtained keypoints with keypoints from the pre-trained model of common type. They shouldn't be worse than those from provided pre-trained model.

**Comment by dreamercv** [comment]
Can you share the pre-trained model with me by e-mail(1367955240@qq.com),because the [pre-trained model link] (https://download.01.org/opencv/openvino_training_extensions/models/human_pose_estimation/checkpoint_iter_370000.pth)  could not  be opened. l want to fine-tuning the model to check the loss.

**Comment by Daniil-Osokin** [comment]
Just make a click of mouse right button on the link and select `Save Link As...`, it should save the checkpoint.
